Resources/Textures/_Trauma/Objects/Materials/materials.rsi/meta.json

```diff
@@ -0,0 +1,242 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/c6e3401f2e7e1e55c57060cdf956a98ef1fefc24 , bear pelt remade by Alekshhh, wood sprite modified by MisterMecky, wood_2 and wood_3 made by MisterMecky based on wood sprite, cardboard sprites made by MisterMecky, bananium, bananium_1 and peel made by brainfood1183 (github) for ss14. Pyrotton sprites are drawn by Ubaser, using the cotton material sprites as a base. Bananium, cardboard, corgihide, cloth, cotton, pyrotton, bones and durathread inhands by mubururu_ (github). Bananium peel's on-head sprite by FlipBrooke, based on the bananaium peel sprite.  General Sprite recolors and diamond sprite made for Traumastation by @Paradoxum on Discord.",
+  "size": {
+    "x": 32,
+    "y": 32
+  },
+  "states": [
+    {
+      "name": "ash"
+    },
+    {
+      "name": "alienhide"
+    },
+    {
+      "name": "bananium"
+    },
+    {
+      "name": "bananium-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "bananium-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "bananium_1"
+    },
+    {
+      "name": "bearpelt"
+    },
+    {
+      "name": "bearpelt_glow",
+      "delays": [
+        [
+          0.2,
+          0.2,
+          0.2,
+          0.2
+        ]
+      ]
+    },
+    {
+      "name": "cardboard"
+    },
+    {
+      "name": "cardboard-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "cardboard-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "cardboard_2"
+    },
+    {
+      "name": "cardboard_3"
+    },
+    {
+      "name": "cathide"
+    },
+    {
+      "name": "cloth"
```

Comment:
This stuff is the source of your test fails. You listed images that don't exist in your RSIs (and don't need to exist).

Go through the meta.jsons and remove sprite states that you did not modify, like cloth. You can see each incorrectly included sprite state in the RSI validation output here: https://github.com/Trauma-Station/Trauma-Station/actions/runs/19019011661/job/54311123235?pr=112